include/verilated_vpi.cpp

```diff
@@ -1315,9 +1382,13 @@ vpiHandle vpi_register_cb(p_cb_data cb_data_p) {
         VerilatedVpiImp::cbTimedAdd(id, cb_data_p, abstime);
         return vop->castVpiHandle();
     }
-    case cbReadWriteSynch:  // FALLTHRU // Supported via vlt_main.cpp
-    case cbReadOnlySynch:  // FALLTHRU // Supported via vlt_main.cpp
-    case cbNextSimTime:  // FALLTHRU // Supported via vlt_main.cpp
+    case cbNextSimTime: {
+        const QData abstime = VL_TIME_Q() + 1;
+        const uint64_t id = VerilatedVpiImp::nextCallbackId();
+        VerilatedVpioTimedCb* const vop = new VerilatedVpioTimedCb{id, abstime};
```

Comment:
```suggestion
        VerilatedVpioTimedCb* const vop = new VerilatedVpioTimedCb{id, abstime};
```

Reply:
Sorry but I don't know what you're suggesting here. Was your intention to remove 
` const QData abstime = VL_TIME_Q() + 1;`  and use
` VerilatedVpioTimedCb* const vop = new VerilatedVpioTimedCb{id, VL_TIME_Q() + 1};`

Reply:
The change is just to use C++ {} instead of () in constructors in new code.